news/templates/news/index.html

```diff
@@ -48,23 +48,6 @@ <h3>{{ article.title }}</h3>
             {% empty %}
                 <h3>No articles currently listed</h3>
             {% endfor %}
-            <div class="pagination">
```

Comment:
Is there pagination that needs removing from the view?

Reply:
@michaeljcollinsuk the only reference to pagination in the view is `paginate_by` which we can use in the project pagination